index.html

```diff
@@ -166,6 +166,11 @@
           <label class="form-check-label" for="pixel-diff-selector">[EXPERIMENTAL] Use the Pixel difference resolution selector (reloads player)</label>
         </div>
 
+        <div class="form-check">
+          <input id=network-info type="checkbox" class="form-check-input">
+          <label class="form-check-label" for="network-info">[EXPERIMENTAL] Use networkInfo API for bandwidth estimations (reloads player)</label>
```

Comment:
We probably don't need `EXPERIMENTAL` here, unless we want to prefix the option.